sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.catalyst._
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.execution.command.ShowTablesCommand
-import org.apache.spark.sql.internal.{SessionState, SharedState, SQLConf}
+import org.apache.spark.sql.internal.{SQLConf, SessionState, SharedState}
```

Comment:
revert; this will likely fail spark scalastyle

Reply:
will revert it. but curious as to why the change in order of names cause scalastyle failure? something which I don't know....?

Reply:
spark/scalastyle-config.xml has an ImportOrderChecker specification that I believe will flag this ordering as issue. I had tried matching the IDEA formatting while back but this ordering within a single import was something that was unable to change. So IDEA's codeStyleSettings.xml we use is incompatible with Spark scalastyle in that respect (maybe can be done now in later IDEA versions).